myuw/templates/base.html

```diff
@@ -152,54 +141,61 @@ <h2 class="sr-only" id="app_messages_desktop_label">MyUW Announcements</h2>
         <div class="myuw-body">
             <div class="myuw-wrapper">
 
-                <!-- app_navigation -->
-                {% block app_navigation %}
-                <div id="menu_container" class="myuw-menu" role="navigation">
+                <div class="myuw-nav-col">
 
-                    <ul>
+                    <!-- app_navigation -->
+                    {% block app_navigation %}
+                    <div id="menu_container" class="myuw-menu" role="navigation">
```

Comment:
is class myuw-menu still defined somewhere in a LESS file? to me it looked like that got changed to a different class name in bolilerplate.less..

Reply:
Yup, in the boilerplate there were two places it was defined. 

1. It was being used to set up the grid using 'display: flex' (flex boxes). line 50-60ish. I wrapped myuw-menu in another div called "myuw-nav-col" to create a left hand column which the nav and the welcome message could sit in. So on line 60, myuw-menu became myuw-nav-col as far as the grid is concerned.

2. myuw-menu was also defined lower in boilerplate.less. This portion does more of the menu styling and the responsive stuff for mobile vs. desktop. That hasn't changed and is still there. Line 341.

Reply:
got it. just didn't see that when i searched boilerplate.less. it may have been collapsed code.